src/test/java/homeworks/Homework19A.java

```diff
@@ -0,0 +1,44 @@
+package homeworks;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19A extends MyBaseTest {
```

Comment:
Please add the `class MyBaseTest `to the branch  Homework19

I can't run the test

![image](https://user-images.githubusercontent.com/104761482/201281183-a593794c-c762-48e9-8796-3da2d08656df.png)

![image](https://user-images.githubusercontent.com/104761482/201281238-fd35659d-5918-494e-816f-e9dafb1e9f08.png)


![image](https://user-images.githubusercontent.com/104761482/201281204-710920ce-758d-42aa-9650-b739b76fa71a.png)